src/dpos-user.ts

```diff
@@ -394,20 +420,62 @@ export class DPOSUser {
     }
     signature = pendingReceipt.oracleSignature
 
+    log('Got receipt', pendingReceipt)
     return CryptoUtils.bytesToHexAddr(signature)
   }
 
   private async withdrawCoinFromRinkebyGatewayAsync(
     amount: BN,
     sig: string
   ): Promise<ethers.ContractTransaction> {
+    const hash = await this.createWithdrawalHash(amount)
+    log('Receipt hash:', hash)
+
+    if (this._version === 2) {
+      // Ugly hack to extract the 'mode' bit from the old signature format - if it's still used (68 = 66 + 2, where 2 is the 0x)
+      sig = sig.length === 134 ? '0x' + sig.slice(4) : sig
```

Comment:
Not clear to me where the 134 nibbles come from... `'0x' + 66 * 2`?

Reply:
66 bytes are represented as a string of length 132
0x is also 2 characters

(signature of 66 bytes) + (0x) = 66 * 2 + 2